api/apps/v1alpha1/nemo_evaluator_types.go

```diff
@@ -827,6 +835,27 @@ func (n *NemoEvaluator) GetServiceMonitorAnnotations() map[string]string {
 	return NemoEvaluatorAnnotations
 }
 
+func (n *NemoEvaluator) GetSecretParams(secretValue string) *rendertypes.SecretParams {
```

Comment:
better to get input param as map[string][string] and populate all key-value pairs in the data field.

Reply:
Fixed